### DISCUSSION TITLE
Sharing env variables with collaborators

### DISCUSSION THREAD

**Discussion by viveleroi** [vercel/community#5537] [Feature Question] [answered]
### Question

Trying Vercel and vercel/postgres and I'm trying to understand how this should work for teams. I've connected my app to a vercel app that uses postgres but I assume I would have to share those `.env` variables with collaborators so they can use the same pipeline I do? I know that .env content is usually not meant to be shared, and I'd commit these values in the `.env.example` file, but I'm making sure I'm not misunderstanding collaboration. I haven't found docs that explain that part.

**Comment by amyegan** [comment, marked as answer]
Hey @viveleroi. The env vars would be accessible to other members of your team unless you moved them to a more restricted role. 

https://vercel.com/docs/accounts/team-members-and-roles/access-roles
